cicd/connection_monitor.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+while true; do
+    clear
+    echo "=== Анализ соединений $(date) ==="
```

Comment:
Consider translating Russian comments and strings for consistency if this script will be used in an English codebase.